projects/Flarum/default.nix

```diff
@@ -1,4 +0,0 @@
-{pkgs, ...}: {
-  packages = {inherit (pkgs) flarum;};
-  nixos.modules.services.flarum = ./service.nix;
```

Comment:
That should be `nixos.modules.services.flarum = null` which means "the module exists, but not in NGIpkgs".